src/helpers/divideByFilteredString.js

```diff
@@ -0,0 +1,26 @@
+export const divideByFilteredString = (inputText, searchedText) => {
```

Comment:
Did you try to find the solution for your case ? :) 
I think, that this function needs some tests :)